rasa/core/channels/console.py

```diff
@@ -89,8 +91,10 @@ def get_cmd_input(button_question: questionary.Question) -> Optional[Text]:
 
     url = "{}/webhooks/rest/webhook?stream=true&token={}".format(server_url, auth_token)
 
+    # Define timeout to not keep reading in case the server crashed in between
+    timeout = ClientTimeout(10)
```

Comment:
I think we should move the `10` to an ALL_CAPS variable on the top, so that we can give this constant a name that properly describes it.

Reply:
done